lib/kubeclient_next/client/api_builder.rb

```diff
@@ -27,8 +27,11 @@ def build!
         resource_descriptions.each do |resource_description|
           next if resource_description.subresource?
 
+          define_create_resource(client, rest_client, resource_description)
           define_get_resources(client, rest_client, resource_description)
           define_get_resource(client, rest_client, resource_description)
+          define_update_resource(client, rest_client, resource_description)
+          define_delete_resource(client, rest_client, resource_description)
```

Comment:
We actually need to check if the resource even exposes these verbs to the client...